src/Windowing/OpenToolkit.Windowing.Desktop/NativeWindow.cs

```diff
@@ -367,8 +361,6 @@ public unsafe int X
             get => Location.X;
             set
             {
-                _location.X = value;
-
                 Glfw.SetWindowPos(WindowPtr, value, _location.Y);
```

Comment:
I know this wasn't writte by you, but you just looked at it, you might know why it has _location.Y and not _location.X?

Reply:
Because it does set value, and keeps location.Y as it was beforehand